app/src/components/pages/appointment.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Query from 'devextreme/data/query';
+import { formatDate } from 'devextreme/localization';
+import { classData } from './data.js';
+
+//Reference: https://js.devexpress.com/Demos/WidgetsGallery/Demo/Scheduler/CustomTemplates/React/Light/
+
+/**
+ * Grab data and formate it for DevExtreme Scheduler
+ */
+
+export function transferdata(data) {
+  var newdata = [];
+  data.map(function (item) {
+    var newitem = {};
+    newitem.courseId = item.CourseID.code;
+    newitem.teacherId = item.professors[0].id;
+    newitem.text = item.CourseID.subject + item.CourseID.code;
+    newitem.startDate = new Date(item.startDate + 'T' + item.meetingTimes[0].StartTime);
+    newitem.teachername = item.professors[0].username;
+    newitem.endDate = new Date(item.startDate + 'T' + item.meetingTimes[0].EndTime);

```

Comment:
There should be a cleaner way to convert these date strings to an ISO string (which is what I presume is accomplished by concatenating the 'T'). Because the app will have a lot of complexities and required functionality involving dates and times, we should look into a library to manage this such as moment.js or whatever is considered industry standard right now.